packages/terralib/tests/postgis/basic/TerraLib.lua

```diff
@@ -1717,38 +1716,38 @@ return {
 		File(toData.file):deleteIfExists()
 
 		tl:saveLayerAs(proj, layerName2, toData, overwrite)
-		unitTest:assert(File(toData.file):exists()) -- SKIP
+		unitTest:assert(File(toData.file):exists())
 
 		-- OVERWRITE AND CHANGE SRID
 		toData.srid = 4326
 		tl:saveLayerAs(proj, layerName2, toData, overwrite)
-		local layerName3 = "SHP"
+		local layerName3 = "PG2SHP"
 		tl:addShpLayer(proj, layerName3, File(toData.file))
 		local info3 = tl:getLayerInfo(proj, proj.layers[layerName3])
-		unitTest:assertEquals(info3.srid, toData.srid) -- SKIP
+		unitTest:assertEquals(info3.srid, toData.srid)
 
 		-- OVERWRITE POSTGIS AND CHANGE SRID
 		local info2 = tl:getLayerInfo(proj, proj.layers[layerName2])
-		unitTest:assertEquals(info2.srid, 29191.0) -- SKIP
+		unitTest:assertEquals(info2.srid, 4019.0)
 		pgData.srid = 4326
 		tl:saveLayerAs(proj, layerName1, pgData, overwrite)
 		info2 = tl:getLayerInfo(proj, proj.layers[layerName2])
-		unitTest:assertEquals(info2.srid, 4326.0) -- SKIP
+		unitTest:assertEquals(info2.srid, 4326.0)
 
 		-- GEOJSON
 		toData.file = "postgis2geojson.geojson"
 		toData.type = "geojson"
 		File(toData.file):deleteIfExists()
 		tl:saveLayerAs(proj, layerName2, toData, overwrite)
-		unitTest:assert(File(toData.file):exists()) -- SKIP
+		unitTest:assert(File(toData.file):exists())
 
 		-- OVERWRITE AND CHANGE SRID
 		toData.srid = 4326
 		tl:saveLayerAs(proj, layerName2, toData, overwrite)
-		local layerName4 = "GJ"
+		local layerName4 = "PG2GJ"
 		tl:addGeoJSONLayer(proj, layerName4, File(toData.file))
 		local info4 = tl:getLayerInfo(proj, proj.layers[layerName4])
-		unitTest:assertEquals(info4.srid, toData.srid) -- SKIP
+		unitTest:assertEquals(info4.srid, toData.srid)
```

Comment:
TerraLib works with srid, so the facade keep its definition for decrease the system gap. It does not impacts for TerraME users.